.github/workflows/rockspec.template

```diff
@@ -0,0 +1,35 @@
+local repo_url = '$repo_url'
+
+rockspec_format = '3.0'
+package = '$package'
+version = modrev ..'-'.. specrev
+
+description = {
+  summary = '$summary',
+  labels = $labels,
+  homepage = '$homepage',
+  $license
+}
+
+dependencies = {
+	'lua >= 5.1',
+  'nui.nvim',
+  'nvim-web-devicons',
+  'plenary.nvim',
+}
+
+external_dependencies = {
+	GIT2 = {
+		library = 'git2',
+	}
+}
```

Comment:
Can we add GPGme as optional dependencies?

Reply:
See my previous response